### ISSUE TITLE
1.9: podman is ignoring --cgroup-manager on command line

### ISSUE THREAD

**Issue by edsantiago** [containers/libpod#5924]
```console
$ cat /etc/containers/containers.conf
[engine]
cgroup_manager = "systemd"
events_logger = "journald"

$ podman --cgroup-manager=cgroupfs  ps
WARN[0000] The cgroupv2 manager is set to systemd but there is no systemd user session available
WARN[0000] For using systemd, you may need to login using an user session
WARN[0000] Alternatively, you can enable lingering with: `loginctl enable-linger 1000` (possibly as root)
WARN[0000] Falling back to --cgroup-manager=cgroupfs
WARN[0000] The cgroupv2 manager is set to systemd but there is no systemd user session available
WARN[0000] For using systemd, you may need to login using an user session
WARN[0000] Alternatively, you can enable lingering with: `loginctl enable-linger 1000` (possibly as root)
WARN[0000] Falling back to --cgroup-manager=cgroupfs
CONTAINER ID  IMAGE  COMMAND  CREATED  STATUS  PORTS  NAMES
```
If I edit (as root) `containers.conf` such that `cgroup_manager = "cgroupfs"`, and run podman (rootless) with an override:
```console
$ podman --cgroup-manager=systemd  ps
CONTAINER ID  IMAGE  COMMAND  CREATED  STATUS  PORTS  NAMES
```
(i.e. no warning messages).

podman-1.9.0-2.module+el8.2.1+6381+7584c8ba

I do not have `libpod.conf` anywhere.

**Comment by rhatdan** [comment]
Are you running rhel8 in cgroup V2 mode?

**Comment by edsantiago** [comment]
No; this is a standard rhel8. [pause] Just confirmed:
```console
# stat -f -c "%T" /sys/fs/cgroup/
tmpfs
```

**Comment by vlastoun** [comment]
I have a similar issue with manjaro:

```
~ » podman -v                                                                                                                          
WARN[0000] The cgroupv2 manager is set to systemd but there is no systemd user session available 
WARN[0000] For using systemd, you may need to login using an user session 
WARN[0000] Alternatively, you can enable lingering with: `loginctl enable-linger 1000` (possibly as root) 
WARN[0000] Falling back to --cgroup-manager=cgroupfs    
podman version 1.9.0
```

**Comment by rhatdan** [comment]
This will be fixed in podman 1.9.1

**Comment by mheon** [comment]
1.9.1 is definitely out, closing as such. Feel free to re-open if this is still happening.
